www/docs/en/dev/guide/platforms/android/lifecycle.md

```diff
@@ -80,14 +150,20 @@ The payload for the `resume` event adheres to the following format:
 {
     action: "resume",
     pendingResult: {
-        pluginServiceName: <name of the plugin e.g. "Camera">,
-        pluginStatus: <description of the result's status (see below)>,
-        result: <argument(s) that would have been given to the callback>
+        pluginServiceName: string,
+        pluginStatus: string,
+        result: any
     }
 }
 ```
 
-The possible plugin statuses in the `pendingResult` field include the following values:
+The fields of that payload are defined as follows:
+
+* `pluginServiceName`: The name of the plugin return the result (e.g. "Camera"). This can be found in a plugin's plugin.xml
+* `pluginStatus`: The status of the plugin call (see below)
+* `result`: Whatever the result of the plugin call is
+
+The possible values for `pluginStatus` in the `pendingResult` field include the following:

```

Comment:
I guess there are two buckets - "OK" and everything else which implies an error - correct?